ramls/examples/uc/uc_credentials_get_200_response.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "ucCredentials",
+  "attributes": {
+    "exist": true
```

Comment:
Is the attribute called "exist" or "isPresent"

Reply:
Thanks! Fixed it.